geomesa-tools/bin/geomesa

```diff
@@ -35,15 +30,48 @@ if [ -z "$GEOMESA_HOME" ]; then
     export GEOMESA_HOME="$bin"
     export PATH=${GEOMESA_HOME}/bin:$PATH
     echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
+}
+
+GEOMESA_OPTS="-Duser.timezone=UTC"
+GEOMESA_DEBUG_OPTS="-Xmx8192m -XX:MaxPermSize=512m -XX:-UseGCOverheadLimit -Xdebug -Xnoagent -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=9898"
+GEOMESA_CP=""
+JAVA_LIBRARY_PATH=""
+
+
+
+# Define GEOMESA_HOME and update the PATH if necessary.
+if [ -z "$GEOMESA_HOME" ]; then
+setGeoHome
 else
-    echo >&2 "Using GEOMESA_HOME $GEOMESA_HOME"
+    echo >&2 "Using GEOMESA_HOME as set: $GEOMESA_HOME"
+    if [[ $1 = configure ]]; then
+        read -p "Is this intentional? Y\n " -n 1 -r
+        if [[  $REPLY =~ ^[Nn]$ ]]; then
```

Comment:
[picky] probably safer to check for yes, and assume other things are no. Also, checking for (y|yes) (ignore case) would be nice.